assets/javascripts/views/search/search_scope.coffee

```diff
@@ -100,6 +101,12 @@ class app.views.SearchScope extends app.View
         $.stopEvent(event) if @doc
     return
 
+  onTextInput: (event) =>
```

Comment:
> Are all keyboard events broken in Android? Does resetting the scope with backspace work? (L92)

Backspace properly sends a keyCode of 8 in the onKeydown event, all other buttons return a keyCode of 229.

> Would something like this work?
>
> ```coffeescript
> onTextInput: (event) =>
>   @onKeydown(event)
> ```

Unfortunately not. The input event always contains 0 for `event.which` on Chrome on Android.

> I'm also worried that this could break other browsers, since the code would run twice (once for the `keydown` event, and another time for the `textInput` event).

I did some more testing on different browsers and it seems that the issue is specific to Chrome on Android. I can confirm that both Firefox on Android and Opera on Android do not have this bug. I'll make a change so it only runs the new event listener on Chrome for Android, and make it so that it doesn't run the entire onKeydown listener on Chrome for Android, but only the part that is necessary for it to function correctly.